docs/examples/models_dynamic_validators.py

```diff
@@ -0,0 +1,18 @@
+from pydantic import create_model, validator
+
+
+def username_alphanumeric(cls, v):
+    assert v.isalnum(), 'must be alphanumeric'
+    return v
+
+
+validators = {
+    'username_validator':
+    validator('username')(username_alphanumeric)
+}
+
+WithValidatorModel = create_model(
+    'WithValidatorModel',
+    username=(str, ...),
+    __validators__=validators
+)
```

Comment:
Looks good, please could you add an example of this being used, e.g. a valid case and an error.

Reply:
👍 Thanks for the feedback.